### ISSUE TITLE
Disable Microsoft Office updates

### ISSUE THREAD

**Issue by JonathanPitre** [EUCweb/BIS-F#96]
Microsoft Updates updates should be disabled during the sealing process to avoid updates from running on production machines. The installed version Office should be detected and the registry values to disable Office updates should be added automatically.

https://social.technet.microsoft.com/Forums/en-US/9124a98f-50c8-4bde-8104-5d18200a31b4/office-365-how-to-temorarily-disable-the-update-feature-for-all-users?forum=Office2016ITPro

**Comment by matthias-schlimm** [comment]
Good point, but if we stop the Windows Update Service during preparation, I think this will help too ?
